src/its_preselector/web_relay.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABC, abstractmethod
+
+
+class WebRelay(ABC):
```

Comment:
Is this reusable in `Preselector`?

Reply:
I figured in theory there could be some other control mechanism used within Preselectors, but went ahead and created an abstract WebRelay class with the notion that there could be others. Down the road we could meke the WebRelayPreselector broadly use the WebRelay but I don't think it is worth it at this point.

Reply:
Gotcha. Good to have the support for the future as more preselectors eventually become supported.